reference/unity_to_godot.rst

```diff
@@ -0,0 +1,177 @@
+|                   |                                                                                   | | GDScript                                                                                                     |
+|                   |                                                                                   | | Integrated debugger                                                                                          |
++-------------------+-----------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+
+
+
+The editor
+----------
+
+Godot Engine provides a rich-featured editor that allows you to build your games. The pictures below display both editors with colored blocks to indicate common functionalities.
+
+.. image:: /img/unity-gui-overlay.png
+.. image:: /img/godot-gui-overlay.png
+
+
+Note that Godot editor allows you to dock each panel at the side of the scene editor you wish.
+
+As you can see, there are some similarities between both editors: first of course, the scene which allows you to display and organize your game elements on screen. Second, the inspector which allows you to edit the parameters of the selected object. Also, the Hierarchy panel in Unity finds its correspondance with Godot's Scene tree panel, although we will see in the next section the differences between how Unity and Godot manage scenes and scene elements.
+
+Unity and Godot both directly use the folder architecture you define for your project. There is no real good solution, and every folder architecture will work for both engines. You can thus continue with your current way to organize your project. However, we have some advices to share a bit further, in order to match your project architecture and Godot scene system.
+
+To finish, the toolbar at the top of the screen is also similar between Unity and Godot. However, Godot provides some new buttons aside the classic "Play", "Pause" and "Stop" buttons. These buttons are introduced in the `Toolbar <introduction/editor#toolbar>`_ section as they a bit out of the scope of this page.
+
+The scene system
+----------------
+
+This is the most important difference between Unity and Godot, and actually the favourite feature of most Godot users.
+
+Unity's scene system generally consist in embedding all the required assets in a scene, and link them together by setting components and scripts to them. 
+
+Godot's scene system is different: it actually consists in a tree made of nodes. Each node serves a purpose: Sprite, Mesh, Light... Basically, this is similar to Unity scene system. However, each node can have multiple children, which make each a subscene of the main scene. This means you can compose a whole scene with different scenes, stored in different files.
```

Comment:
I would simplifly it like:

A scene in Godot is simple, it consists of a tree made of nodes. A node could be seen as a GameObject with one component and one behavior and, as an extra limitation, scenes can only have one root node.

While this restriction can be seen as arbitrary at first, it makes the whole system a lot more flexible and powerful for the following reasons:

- One function per node makes scenes more descriptive at a glance. It's very easy in Godot for someone who didn't create a scene to understand how it works by just taking a quick look at it. 
- It allows for a divide and conquer approach to create game content like no other tool. Scenes can be edited separately and then instanced and nested at any level. 
- Even inheriting a scene is possible. Make a base enemy scene with common functions (targeting, explosion, hitpoint display, etc), and inherit it to create every kind of enemy. Import entire scenes from your favorite 3D DCC, then inherit them to do local modifications while preserving the original for re-import.
-Teamwork is easier than with any other game engine, as it's very easy to divide a project in scenes. Artist, programmer, game designer and level designer will work most of the time in their own scenes without stepping on each other's toes.